docs/sources/dashboards/export-import/grafana2-3.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Import and share with Grafana 2.x or 3.0"
+keywords = ["grafana", "dashboard", "documentation", "export", "import"]
+aliases = ["/docs/grafana/latest/reference/export_import/"]
+weight = 800
++++
+
+# Import and share with Grafana 2.x or 3.0
```

Comment:
Is this specific to these two versions of Grafana, or is this functionality still current? If for two old versions, then let's get rid of it. Otherwise, let's rename it.

Reply:
I think grafana 3 or later. Will check.

Reply:
I do believe we can skip this section :smile: but we should probably ask @grafana/grafana-frontend-platform as well

Reply:
@dprokop wdyt?